trajectory_tracker/src/trajectory_tracker.cpp

```diff
@@ -389,7 +393,9 @@ void TrackerNode::cbOdometry(const nav_msgs::Odometry::ConstPtr& odom)
         odom_to_robot.inverse(),
         odom->header.stamp, odom->header.frame_id);
 
-    control(robot_to_odom, prediction_offset, dt);
+    const double odom_linear_vel = std::hypot(odom->twist.twist.linear.x, odom->twist.twist.linear.y);
```

Comment:
As `trajectory_tracker` doesn't `controll twist.linear.y`, I think `odom_linear_vel` is better to be `twist.linear.x`.

Reply:
Done